gremlin-go/docker-compose.yml

```diff
@@ -24,6 +24,8 @@ services:
     build:
       context: ../
       dockerfile: gremlin-go/Dockerfile
+      args:
+        NEO4J_VERSION: 3.5.3
```

Comment:
hardcoding to a previous version isn't going to work. tests should run against the most recent build (i.e. whatever is in the current top-level pom.xml).